### ISSUE TITLE
Alias vs mistaken identity

### ISSUE THREAD

**Issue by tlongers** [security-force-monitor/sfm-research-handbook#40]
`Operación Coordinada Chihuahua` is listed as an alias of `Operación Conjunta Chihuahua`. I think this is a correct application of the `organization:alias` rules, as the two operations have been confused.  We need a specific field to note mistaken identities.

**Comment by tonysecurityforcemonitor** [comment]
Good point that there is a difference between other ways to refer to an organization/alias and mistakes in identity. Question about resources to make this happen (might be negligible) and where that lives on WWIC (I think up top with other name information makes sense, but could also see having section somewhere else as well)
